src/DynamoCore/Graph/Workspaces/WorkspaceModel.cs

```diff
@@ -1333,6 +1334,21 @@ internal set
             }
         }
 
+        /// <summary>
+        /// PolyCurve normal and direction behavior has been made predictable in Dynamo 3.0 and has therefore changed. 
+        /// This reflects whether legacy (pre-3.0) PolyCurve behavior is selected either in preference settings or in the workspace.
+        /// A workspace setting if exists, overrides the default preference setting. 
+        /// </summary>
+        internal bool EnableLegacyPolyCurveBehavior
+        {
+            get { return enableLegacyPolyCurveBehavior; }
+            set
+            {
+                enableLegacyPolyCurveBehavior = value;
+                WorkspaceEvents.OnWorkspaceSettingsChanged(enableLegacyPolyCurveBehavior);
```

Comment:
Would it be useful to check if the value actually changed before calling the event?

Reply:
@pinzart90 I decided not to go ahead with your suggestion in the end as I was seeing a regression because of this.